apps/web/src/components/workflow/preview/in-app/Content.styles.ts

```diff
@@ -69,6 +69,14 @@ export const NotificationTextStyled = styled.div<{ isExampleNotification: boolea
   }}
 `;
 
+export const SubjectTextStyled = styled.p`
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 20px;
+  margin: 0;
```

Comment:
```suggestion
  font-size: 16px;
  font-style: normal;
  font-weight: 600;
  line-height: 20px;
  margin: 0;
```

This doesn't conform to the Design System value

Reply:
Good catch, thanks!